test/test_reset.py

```diff
@@ -1,89 +1,29 @@
-from opensquirrel import Circuit, CircuitBuilder
-
-
-def test_reset() -> None:
-    qc = Circuit.from_string(
-        """
-        version 3.0
-
-        qubit[2] q
-        bit[2] b
-        H q[0]
-        CNOT q[0], q[1]
-
-        reset q[0]
-        b = measure q
-        """,
-    )
-    assert (
-        str(qc)
-        == """version 3.0
-
-qubit[2] q
-bit[2] b
-
-H q[0]
-CNOT q[0], q[1]
-reset q[0]
-b[0] = measure q[0]
-b[1] = measure q[1]
-"""
-    )
-
+import pytest
 
-def test_reset_sgmq() -> None:
-    qc = Circuit.from_string(
-        """
-        version 3.0
-
-        qubit[4] q
-
-        H q[0]
-        H q[1:2]
-        reset q[2:3]
-        H q[3]
-        reset q[0:1]
-        """,
-    )
-    assert (
-        str(qc)
-        == """version 3.0
+from opensquirrel import Circuit, CircuitBuilder
+from opensquirrel.ir import Reset
 
-qubit[4] q
 
-H q[0]
-H q[1]
-H q[2]
-reset q[2]
-reset q[3]
-H q[3]
-reset q[0]
-reset q[1]
-"""
-    )
+@pytest.mark.parametrize(
+    ("cqasm_string", "expected_result"),
+    [
+        ("version 3.0; qubit[2] q; reset q[1]; reset q[0]", "version 3.0\n\nqubit[2] q\n\nreset q[1]\nreset q[0]\n"),
+        (
+            "version 3.0; qubit[4] q; reset q[2:3]; reset q[1, 0]",
+            "version 3.0\n\nqubit[4] q\n\nreset q[2]\nreset q[3]\nreset q[1]\nreset q[0]\n",
+        ),
```

Comment:
Same same but different.

```suggestion
        (
            "version 3.0; qubit[2] q; reset q[1]; reset q[0]",
            "version 3.0\n\nqubit[2] q\n\nreset q[1]\nreset q[0]\n"
        ),
        (
            "version 3.0; qubit[4] q; reset q[2:3]; reset q[1, 0]",
            "version 3.0\n\nqubit[4] q\n\nreset q[2]\nreset q[3]\nreset q[1]\nreset q[0]\n",
        ),
```

Reply:
I accepted your changes, but `ruff format` prescribes the old form...

Reply:
Maybe worth using `# fmt: skip`?